backend/services/interfaces/testSessionService.ts

```diff
@@ -41,33 +41,24 @@ export interface TestSessionResponseDTO {
    * the result of the test session
    * there should be one entry here per student
    * */
-  results?: ResultResponseDTO[];
+  results?: ResultDTO[];
```

Comment:
Hmmm... so I think we had some context for why we had kept Response and Request DTOs separate (despite their similarities), see [here](https://github.com/uwblueprint/jump-math/pull/25#discussion_r917479900). I think Mahad makes a good point about keeping them separate for the sake of modularity!

If you're interested, you can read a bit more about this [here](https://stackoverflow.com/questions/44349199/reusing-dto-for-various-request-response-types-vs-explicitness-of-what-is-requir)! (Although, the link talks about separating common elements into some sort of base class, but I think that's overkill for our use case here)